testapps/ngx/.env.development.local

```
@@ -0,0 +1,2 @@
+VITE_APP_TOLGEE_API_URL=https://app.tolgee.io
+VITE_APP_TOLGEE_API_KEY=tgpak_gm4dooc7omztg5lrmjtgiy3kmfzdentnm5xw22lpm5zda5rwme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_TOLGEE_API_KEY key should go before the VITE_APP_TOLGEE_API_URL key

(UnorderedKey)